pkg/features/embedded/grub-config/etc/elemental/grub.cfg

```diff
@@ -57,6 +57,12 @@ function set_loopdevice {
   loopback ${volume} ${1}
 }
 
+## Sources bootargs from the current volume
+function source_bootargs {
+  source (${volume})/etc/cos/bootargs.cfg
```

Comment:
Isn't this failing if the file is not there? 🤔

Reply:
Not when I tried it.. I was going to check if the file existed first but seems like it's not needed :shrug: